backend/inspirehep/search/api.py

```diff
@@ -323,6 +358,16 @@ class Meta:
         index = "records-journals"
         doc_types = "_doc"
 
+    def normalize_title(self, journal_title):
+        normalized_journal_title = journal_title
+        hits = self.query("match", lowercase_journal_titles=journal_title).execute()
+        if hits:
+            try:
+                normalized_journal_title = hits[0].short_title
+            except (AttributeError, IndexError):
+                LOGGER.info("Failed to normalize journal title in", result=hits[0])
```

Comment:
why would this happen? `short_title` is required by the schema. I think this is a real exception, not something to basically ignore by logging as `info`.

Reply:
👍  tbh I just copy paste the code from https://github.com/inspirehep/inspire-next/blob/d6898dfbe061977208a7a69be0d1e7e0c46a8a7d/inspirehep/utils/normalizers.py#L31. I should have refactored it.